nengo_spa/actions.py

```diff
@@ -26,10 +30,15 @@ class Parser(object):
         'reinterpret': Reinterpret,
         'translate': Translate}
 
-    def __init__(self, vocabs=None):
-        if vocabs is None:
-            vocabs = {}
-        self.vocabs = vocabs
+    def __init__(self, locals_=None):
```

Comment:
Appending an underscore seems like a pretty common think to do in these cases (especially as `locals_` is intended to be `locals()`). I'm pretty sure I have seen similar things in other code bases. But I'm open to changing it if that's the consensus.

I feel @tbekolay might dislike the underscore too?